_sass/_variables.scss

```diff
@@ -26,7 +26,7 @@ $teal: #20c997 !default;
 $cyan: #17a2b8 !default;
 
 $primary: #006931 !default;
-$logo-color: #1e9348 !default;
+$logo-color: #1a8440 !default;
```

Comment:
This is a brand color. We have T-shirts and mugs in this green heheh.  Could we accomplish the goal by tuning a different color?